packages/drivers/form/src/actions.js

```diff
@@ -0,0 +1,15 @@
+export default key => state => (name) => {
+  const set = type => (values = {}) => state[type].addOrUpdate({ [key]: name, ...values })
+
+  return ({
+    init: set('values'),
```

Comment:
I think `init` is useless now since `get` take care of undefined AND `set` is doing the same thing

https://github.com/alakarteio/k-ramel/pull/191/files#diff-fadb9994e60d3d9b44c000988a6f31e7R12

Reply:
It's to understanding but ok.